### DISCUSSION TITLE
For any post show the tags (linking to list of posts with that tag) just below the title

### DISCUSSION THREAD

**Discussion by SageofPHY6Paths** [MrGreensWorkshop/MrGreen-JekyllTheme#164] [Feature requests]
### Is this enhancement or feature is related to a problem?

This will be useful. When reading a post a reader might want to see what tags it is related to and he can click on that see similar posts.

On the blog page https://jekyll-theme-mrgreen-demo.mrgreensworkshop.com/tabs/blog/ it is not needed since tags are already there in the up navigation.

### Describe the solution you'd like

I am not sure exactly. Normally Jekyll blogs are written such that each tag has its on page and tags below the title will be linked to that.

But here it is interactive kind of thing. All tags are there in the same page. They don't even have some bookmark in that page like `#tagname`. So, not sure how this can be done.

**Comment by SageofPHY6Paths** [comment]
![image](https://user-images.githubusercontent.com/92218586/210180158-9b1a398c-00c3-41ea-8625-0079db0a6eef.png)

Tags with links either after or before the number of comments will be fine.

**Comment by MrGreensWorkshop** [comment]
I thought I already added that feature, but I couldn't find it in the settings. I was planning to add that, but I think I dismissed that feature for the first release and forgot about it.

This is where tab links are generated. (_includes/post-list/tab-links.html)

Btw, if you want to add yourself, check this file: _includes/post_common/post-info-html.html
